types/fetch_method.pp

```diff
@@ -0,0 +1,11 @@
+# @summary A supported fetch method for poudriere
+type Poudriere::Fetch_method = Enum[
+  'git',
```

Comment:
git can have almost the same +es as svn:

```
git, # sadly the default with this is the git protocol
git+http, # bad
git+https,
git+ssh,
```

i also have this pr https://github.com/freebsd/poudriere/pull/813

Reply:
Yes, git is becommind a thing I guess with the switch to the git VCS.  I adjusted the list accordingly, thanks!